cbrain_task/civet/bourreau/civet.rb

```diff
@@ -331,8 +352,10 @@ def cluster_commands_single(options_to_ignore={}) #:nodoc:
       args += "-reset-from #{reset_from.bash_escape} "
     end
 
-    mincfiles_dir = "mincfiles_input"
-    civet_command = "CIVET_Processing_Pipeline -prefix #{prefix.bash_escape} -source #{mincfiles_dir} -target civet_out -spawn #{args} -run #{dsid.bash_escape}"
+    mincfiles_dir  = "mincfiles_input"
+    civet_command  = "CIVET_Processing_Pipeline -source #{mincfiles_dir} -target civet_out -spawn #{args} "
+    civet_command += "-prefix #{prefix.bash_escape} " if prefix.present?
+    civet_command += "-run #{dsid.bash_escape}"       if dsid.present?
```

Comment:
I am not sure about the behavior of the prog with a missing subject ID... and how woudl CBRAIN even be able to check the outputs?

I think in CBRAIN we should simply force the prefix and the subject ID to be present.